index.html

```diff
@@ -636,12 +636,12 @@ <h1 id="sponsors"><strong>PARTNERS</strong></h1>
             <ul class="list-unstyled">
 
                 <li>
-                    <a href="http://megathon.tech/#">
+                    <a target="_blank" href="http://megathon.tech/#">
```

Comment:
Change the link address to https://www.zomato.com/hyderabad/wich-please-gachibowli